apps-rendering/src/components/Callout/calloutBlock.tsx

```diff
@@ -0,0 +1,44 @@
+// import {renderToString, renderToStaticMarkup} from 'react-dom/server';
+import type { FormField } from '@guardian/apps-rendering-api-models/formField';
+import type { ArticleFormat } from '@guardian/libs';
+import type { FC, ReactElement } from 'react';
+import { renderCalloutDescriptionText } from 'renderer';
+import CalloutForm from './calloutForm';
+import {
+	calloutContainerStyles,
```

Comment:
nit - can we remove `Styles` from the end of this name so its consistent with the others? its also implied by the import so its a nicer naming pattern in my opinion.

Reply:
For sure - I missed this one!